### PR TITLE
Fix sensor status calls

### PR DESCRIPTION
**Pull request by haslinghuis**
Fix calling `sensor_status` in `update_live_status` in serial_backend so it is no longer needed to call it from MSP message requests as with gps enabled it got called twice within the same timer loop.

Related to https://github.com/betaflight/betaflight/pull/13177

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by asizon — APPROVED** [review]
LGTM!

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
